bloom/generators/debian/templates/rules.em

```diff
@@ -30,7 +30,8 @@ override_dh_auto_configure:
 	dh_auto_configure -- \
 		-DCATKIN_BUILD_BINARY_PACKAGE="1" \
 		-DCMAKE_INSTALL_PREFIX="@(InstallationPrefix)" \
-		-DCMAKE_PREFIX_PATH="@(InstallationPrefix)"
+		-DCMAKE_PREFIX_PATH="@(InstallationPrefix)" \
+		-DCMAKE_BUILD_TYPE=RelWithDebInfo
```

Comment:
Thanks for the reference! I have removed this and will push the update shortly. I did a quick test and the default build type still passes -g to the compiler so the debug symbols are generated.